code/network/multi.cpp

```diff
@@ -493,6 +494,12 @@ void process_packet_normal(ubyte* data, header *header_info)
 {
 	switch ( data[0] ) {
 
+		// this is for helping to diagnose misaligned packets.  The last sensible packet 
+		// is usually the culprit that needs to be analyzed.
+		if (Cmdline_dump_packet_type) {
```

Comment:
On second look, shouldn't this if go outside the switch?

Reply:
Yup...... I wasn't rushing at all

Reply:
The changed version seems a bit extra on white space, but up to you if you want to adjust it imo.